elements/lisk-chain/src/constants.ts

```diff
@@ -28,3 +28,6 @@ export const GENESIS_BLOCK_TRANSACTION_ROOT = EMPTY_HASH;
 
 export const TAG_BLOCK_HEADER = createMessageTag('BH');
 export const TAG_TRANSACTION = createMessageTag('TX');
+
+// TODO: Actual size TBD
+export const MAX_ASSET_DATA_SIZE_BYTES = 64;
```

Comment:
I think genesis block will exceed this value.

Reply:
what could be the safe value until we get the announced value?